### PR TITLE
issue #136 : Fix No-Contact text 

### LINKED ISSUE #136: Fix No Contacts element on team page
The no contact element is not occupying complete space in team page

### PR DESCRIPTION
**Pull request by Haider8**
Fixes #136

### CODE REVIEW

File: src/components/Other/Team.vue [hunk 1]
```diff
@@ -144,4 +144,7 @@ export default {
   color: black;
   text-align: center;
 }
+.contact {
+  font-size: 35px;
```
Comment (iamareebjamal) [inline src/components/Other/Team.vue]: That's quite large. What if we don't show the no contact panel altogether

Comment (Haider8) [reply src/components/Other/Team.vue]: yes, that would also be okay.

**Comment by iamareebjamal** [comment]
Change the PR description to `Fixes #1361 so that issue automatically closes on merging of PR

**Comment by Haider8** [comment]
Done

**Pull request merged**